.github/workflows/codecov.yml

```diff
@@ -19,21 +19,10 @@ jobs:
         python-version: 3.8
     - name: Install dependencies
       run: |
-        python -m pip install --upgrade pip
-        pip install -r requirements.txt
```

Comment:
I love that you got rid of the multiple copies of the ephemerides setup! Rest of the workflow cleanup also looks good. But why removing the `python -m pip install --upgrade pip` here and in the others, that seems a reasonable precaution given how often pip deprecates and randomly breaks?

Reply:
You are right, let's keep it.